src/encoded/static/components/search.js

```diff
@@ -734,6 +764,17 @@ const SeriesComponent = ({ context: result, auditDetail, auditIndicators }, reac
                             phases.push(biosample.phase);
                         }
                     }
+                    if (replicate.library) {
```

Comment:
Would it be better if you do -
`if (!replicate.library) { return; }`
and if this above results in falsey, continue on with the multiple if-statements...